app/javascript/components/products/Products.js

```diff
@@ -13,7 +13,7 @@ const Products = () => {
     fetch('/solidus/api/products.json?page=1&per_page=6', {
       method: 'GET',
       headers: {
-        'Authorization': 'Bearer 75be11a6e0d07b836aa2621c65f8936706499db21f0d0e5f',
+        'Authorization': 'Bearer fa79bc4134b5ffa7d9af1fcb487aa806f710d4b912a73041',
```

Comment:
remove this change, every key is different for everyone

Reply:
Ready!